bedrock/security/urls.py

```diff
@@ -41,4 +41,5 @@
     re_path(r"^known-vulnerabilities/(?P<filename>.*)\.html$", KVRedirectsView.as_view()),
     re_path(r"^(?:announce|advisories)(?:/.*)?/mfsa(?P<pk>\d{4}-\d{2,3})\.html$", OldAdvisoriesView.as_view()),
     path("announce/", OldAdvisoriesListView.as_view()),
+    page("third-party-software-injection", "security/third-party-injection-policy.html"),
```

Comment:
**issue:** `http://localhost:8000/en-US/security/third-party-software-injection/` (with trailing slash) gives me 404 but `http://localhost:8000/en-US/security/third-party-software-injection` is fine 
<img width="827" alt="Screen Shot 2022-04-06 at 10 19 57 AM" src="https://user-images.githubusercontent.com/19650432/161943447-399bdda3-edc0-4aec-b2fb-61d07ffe8da6.png">

<img width="859" alt="Screen Shot 2022-04-06 at 10 19 47 AM" src="https://user-images.githubusercontent.com/19650432/161943487-04e4a3a6-79c8-4a5c-8270-7c2a8d652ed0.png">


**suggestion:** looks like since https://github.com/mozilla/bedrock/pull/11396/files#diff-b0bd8ab9f6d9ff6b64da33b620fafbf836b273a575e8f665d10c0511cf9555a1 the page helper does not automatically add trailing slash and requires one included on the URL pattern